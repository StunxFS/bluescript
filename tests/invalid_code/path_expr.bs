mod my_mod {
    const XXX = 1;

    enum Afk { a }
}

fn main() {
    const ABC = my_mod::XXX; // ok
    const DEF = my_mod::Afk; // error
    const GHI = my_mod::unown; // error
}
