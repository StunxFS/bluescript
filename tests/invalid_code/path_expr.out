tests/invalid_code/path_expr.bs:9:25: error: expected value, found enum `Afk`
tests/invalid_code/path_expr.bs:10:25: error: module `path_expr::my_mod` does not contain a symbol named `unown`
